README.md

```diff
@@ -448,6 +448,29 @@ scale: 0.001 # floating point factor applied to result, e.g. for Wh to kWh conve
 timeout: 30s # error if no update received in 30 seconds
 ```
 
+### SMA/Speedwire (read only)
```

Comment:
Fehlt noch die erwähnung in der plugins section

Reply:
zum index hinzugefügt